host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_5.java

```diff
@@ -336,6 +336,7 @@ protected void readDomainElementAttributes(XMLExtendedStreamReader reader, Model
                         list.add(op);
                         break;
                     }
+                    case  ORGANIZATION: // not in the xsd but let's be forgiving to ease migration
```

Comment:
You've shamed me out of my laziness. ;) A couple lines will address this so I'll add them.